cpg-language-python/src/main/kotlin/de/fraunhofer/aisec/cpg/frontends/python/StatementHandler.kt

```diff
@@ -284,6 +287,13 @@ class StatementHandler(frontend: PythonLanguageFrontend) :
                         recordDeclaration = recordDeclaration,
                         rawNode = s
                     )
+                } else if (language is HasOperatorOverloading && s.name.isKnownOperatorName) {
```

Comment:
It'd be nice have have this automatically...

Reply:
Unfortunatly, this is not that easy without breaking the space-time-continuum. I would leave it as-is for this PR but I will try to come up with a solution.

Reply:
Ok, it would have been nice, but I don't see this as a deal-breaker for the pr -> I'm resolving this conversation.